### PR TITLE
add UK band charts

### LINKED ISSUE #66: add UK band chart
### What would you like to add?

A band chart for the UK

### Is this resource available in a usable format or does it require some work?

Yes, decent charts exist for HF and VHF.

### Additional context

I already have a PR ready for it 🙂

### PR DESCRIPTION
**Pull request by posimagi**
### Description

Add UK band charts.

Fixes #66 

### Type of change

Resource update

### How has this been tested?

N/A

### Checklist

- [x] Issue exists for PR
- [ ] Code reviewed by the author
- [ ] Code documented (comments or other documentation)
- [ ] Changes tested
- [ ] All tests pass (see [`DEVELOPING.md`][0], if it exists)
- [x] [`CHANGELOG.md`][1] updated if needed
- [ ] Informative commit messages
- [ ] Descriptive PR title

[0]: /DEVELOPING.md
[1]: /CHANGELOG.md

### CODE REVIEW

**Comment by posimagi** [comment]
#66 add UK band chart

**Comment by posimagi** [comment]
Images have been compressed with pngquant.

File: resources/bandcharts.1.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
     "nl": ["bandcharts/nl.png", "Netherlands", "Amateur radio bands in the Netherlands", "*This version of this bandplan is no longer hosted on VERNON's website and it's accuracy cannot be guaranteed.*", "[VERNON Global Band Plan van de Telecom Agency, Ministerie van Economische Zaken en Klimaat](https://www.veron.nl/downloads/brochures/)", "🇳🇱"],
     "ph": ["bandcharts/ph.png", "The Philippines", "Amateur radio bands in the Philippines", "", "DW5CD [[PDF]](https://github.com/miaowware/qrm-resources/files/7668695/Philippine.Amateur.Band.Plan.2021.pdf)", "🇵🇭"],
     "ru": ["bandcharts/ru.png", "Russia", "Amateur radio bands in Russia (HF Only)", "", "[Amateur Operator's Union of Russia](https://srr.ru/radiooperatoram/radiochastoty/)", "🇷🇺"],
+    "uk_hf": ["bandcharts/uk_hf.png", "UK (HF)", "Amateur radio bands in the UK (HF Only)", "", "[Online Amateur Radio Community](https://wiki.oarc.uk/_media/reference:uk_ham_bands_chart_-_v0003_-_2_pager.pdf)", "🇬🇧"],
+    "uk_vhf": ["bandcharts/uk_vhf.png", "UK (VHF)", "Amateur radio bands in the UK (VHF Only)", "", "[Online Amateur Radio Community](https://wiki.oarc.uk/_media/reference:uk_ham_bands_chart_-_v0003_-_2_pager.pdf)", "🇬🇧"],
```
Comment (0x5c) [inline resources/bandcharts.1.json]: Use "United Kingdom" instead of "UK" in the short name (second field)

Comment (0x5c) [inline resources/bandcharts.1.json]: The "VHF" page doesn't have anything higher than 70cm. Is there a 3rd chart for higher/other bands? If so we're interested in having it too

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ The version format should be `YYYY.mM.seq`. For example: the *third* version rel
 
 ## [Unreleased]
 
+### Added
+- UK (`uk_hf`, `uk_vhf`) band charts.
```
Comment (0x5c) [inline CHANGELOG.md]: "United Kingdom" here too

**Review by 0x5c — CHANGES_REQUESTED** [review]
Wow those look good! Probably the bandcharts that are the most readable at discord size yet.

While addressing the change request, squash the two commits together to make a single commit. You can do that with `git rebase -i master` while in your branch, only `pick`ing the first one, and `squash`ing the others
Also do that rebase for the bulgarian bandchart PR

File: resources/bandcharts.1.json (hunk 1 above)
Comment (posimagi) [reply resources/bandcharts.1.json]: If there is, I'm unaware of it.

**Review by 0x5c — APPROVED** [review]

**Comment by 0x5c** [comment]
Thanks for contributing these! We'll make a release once the bulgarian charts are merged too.

**Pull request merged**